website/docs/programming-languages.md

```diff
@@ -34,7 +34,6 @@ For an actual example of an issue or pull request adding the above support, plea
 
 | Language | Stop Words (time to contribute: <5 min) | Repository Context (time to contribute: <1 hr) |
 | :------: | :-------------------------------------: | :--------------------------------------------: |
-|  C/C++   |                    🚫                    |                       🚫                        |
```

Comment:
Add C/C++ to `Supported Languages`, and mentioned it will be supported since 0.8.0